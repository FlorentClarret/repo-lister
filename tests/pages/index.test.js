import renderer from 'react-test-renderer'

import App from '../../pages/index.js'

import realData from '../../data'

const getRepos = (data) => Object
  .values(data)

describe('With real data', () => {
  it('Renders without errors', () => {
    renderer.create(<App repos={getRepos(realData)}/>)
  })
})
