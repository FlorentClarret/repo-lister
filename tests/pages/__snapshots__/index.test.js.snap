// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App matches snapshot 1`] = `
Array [
  <nav
    className="navbar navbar-expand-md pt-3 pb-3 navbar navbar-light"
  >
    <a
      className="navbar-brand"
      href="/"
      onClick={[Function]}
    >
      <span
        className="icon ion-md-home mr-1"
      />
      Home
    </a>
  </nav>,
  <div
    className="container-fluid"
  >
    <h2
      className="text-center display-4 mt-5 mb-2"
    >
      List of posts
    </h2>
    <div
      className="card-columns"
    >
      <div
        className="jsx-3555720613 RepoCard"
      >
        <div
          className="Foo card"
        >
          <div
            className="card-body"
          >
            <h4
              className="card-title"
            >
              <div
                className="jsx-3555720613"
              >
                <div
                  className="jsx-3555720613"
                >
                  <a
                    className="jsx-3555720613"
                    href="html_url"
                    onClick={[Function]}
                  >
                    login
                  </a>
                   / 
                  <a
                    className="jsx-3555720613"
                    href="html_url"
                    onClick={[Function]}
                  >
                    name
                  </a>
                </div>
              </div>
            </h4>
            <p
              className="card-text"
            >
              description
            </p>
            <p
              className="gh-stats card-text"
            >
              <span
                className="jsx-2393098220 language-container"
              >
                <span
                  className="jsx-2393098220 color"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                >
                  ‚óè
                </span>
                <span
                  className="jsx-2393098220 text"
                >
                  language
                </span>
              </span>
              <span
                className="jsx-2564559751 issues-container"
              >
                <a
                  className="jsx-2564559751"
                  href="html_url/issues"
                  onClick={[Function]}
                >
                  <span
                    className="jsx-2564559751 icon ion-md-information-circle mr-1"
                  />
                  0
                </a>
              </span>
              <span
                className="jsx-1533896771 pr-container"
              >
                <a
                  className="jsx-1533896771"
                  href="html_url/pulls"
                  onClick={[Function]}
                >
                  <span
                    className="jsx-1533896771 icon ion-md-git-pull-request mr-1"
                  />
                  0
                </a>
              </span>
            </p>
            <button
              className="btn-sm btn btn-secondary"
              onClick={[Function]}
              type={undefined}
            >
              More info
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <footer>
    <div
      className="col-md-offset-3 text-center container"
    >
      <hr
        className="mt-3"
      />
      <p
        className="text-muted small"
      >
        <span>
           Updated 
        </span>
        <time
          className="font-weight-bold"
          data-dateepoch={946684800000}
          data-datetime="2000-01-01T00:00:00.000+00:00"
          title="January 1st 2000, 00:00:00"
        >
          less than 5 seconds ago
        </time>
        <span>
          , built with 
        </span>
        <a
          className="text-muted font-weight-bold"
          href="https://github.com/zeit/next.js"
          onClick={[Function]}
        >
          Next.js
        </a>
        .
      </p>
    </div>
  </footer>,
]
`;
